.github/assets/.template.yaml

```diff
@@ -1,4 +1,8 @@
-cluster_name: '{{RAY_CLUSTER_NAME}}'
+# Note:
+# GitHub Actions workflow will replace all parameters between `{{...}}` with the
+# actual values as determined dynamically during runtime of the actual workflow.
+
+cluster_name: \{{CLUSTER_NAME}}
```

Comment:
If we don't add a `\`, YAML will assume that to be an object, as opposed to a string. Instead of this, I could also wrap the string with quotes explicitly (i.e., `'{{}}'`).

Reply:
I like the second option that you proposed. I'll implement that in a follow up PR.